packages/compass-import-export/src/modules/import.ts

```diff
@@ -245,6 +214,18 @@ export const startImport = () => {
       transform,
     } = importData;
 
+    // Create the error log output file.
+    const userDataPath = getUserDataFolderPath();
+    if (!userDataPath) {
+      throw new Error('cannot access user data for error log generation');
+    }
```

Comment:
Good question, I could see both sides having advantages:
- If we don't break early here a user wouldn't be able to get a full list of import errors that occur. That could be potentially frustrating if errors do occur, a user wouldn't be able to discern what went wrong and what data made it in.
- On the other hand if a user can't import because of a file issue, they wouldn't be able to run into that possible situation, however they're probably more frustrated as they can't import to begin with.

I just updated it to still proceed and add the error as the first error so it's always shown in the modal.